site/content/guides/monitoring-as-code.md

```diff
@@ -2,6 +2,9 @@
 title: Monitoring as code
 description: >-
   The trend of declaring infrastructure as code has been picking up steam over the last few years, offering a way for DevOps teams to transparently manage and scale cloud infrastructure. Why should the way we manage monitoring be any different? In this article, we address this point and illustrate it with a practical example of Monitoring-as-Code on Checkly.
+metatags:
+  title: "Monitoring as Code: Guide to Code Monitoring with Checkly"
+  description: "Discover how Monitoring as Code transforms cloud infrastructure management with a hands-on Checkly case study. Start optimizing your workflow today!"
```

Comment:
Just to double check: Is okay to use "Monitoring as Code" with capital C when we call it "Monitoring as code" around the website?

Reply:
I think this is an example of [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) although it does contain an error for APA title case: 'with' should be capitalized! will fix